### ISSUE TITLE
Add a tutorial entry to explain how to re-evaluate the best trial.

### ISSUE THREAD

**Issue by toshihikoyanase** [optuna/optuna#1616]
Thanks to @nzw0301 , the objective functions accept `FrozenTrial` as their input (See #1503). 
It enables us to re-evaluate the best trial easily as follows:

```python
import optuna

def objective(trial):
    ...
    return accuracy

study = optuna.create_study()
study.optimize(objective, n_trials=100)

print(study.best_trial.value)  # Show the best value.
print(objective(study.best_trial)  # Re-evaluate the objective function.
```

This feature helps users to reproduce their experiments and is worth writing a tutorial entry.

**Comment by nzw0301** [comment]
May I ask how to add a tutorial entry? Tutorial scripts look like unusual python scripts (jupyter?). Is there any guideline to add a tutorial entry?

**Comment by nzw0301** [comment]
I believe that this issue can be closed by #2427!
